packages/accessibility/.eslintrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["../../build/eslint/index.js"],
```

Comment:
@kenotron once this is merged, you'll probably need to add a stop to your script to update these paths

Reply:
In both .eslintrc.json and tslint.json